subsys/net/lib/sockets/sockets_can.c

```diff
@@ -95,23 +95,20 @@ static void zcan_received_cb(struct net_context *ctx, struct net_pkt *pkt,
 	 * which will give the packet to correct net_context(s).
```

Comment:
The commit message `samples: net: sockets: socketCAN: add loopback mode` of the 3rd commit does not seem to be correct here. It talks about sample, but this change is for socket CAN core code.

Reply:
Sorry, this is not supposed to go in.